### PR TITLE
Adding brain test job to cap-release pipeline.

### PR DESCRIPTION
**Pull request by andreas-kupries**
Ticket: https://jira.suse.com/browse/CAP-1247

### CODE REVIEW

File: cap-ci/pipeline.template [hunk 1]
```diff
@@ -275,6 +275,64 @@ jobs:
             {{- end }}
             {{- print $test }}
 
+- name: cf-acceptance-tests-brain-{{$EiriniFlag}}-{{$Backend}}-{{$OptionsFlag}}
```
Comment (viccuad) [inline cap-ci/pipeline.template]: Nitpicking: could the `cf-acceptance-tests-brain` job block go below the `cf-acceptance-tests` one? that way the tasks are ordered as in the pipeline.

**Review by viccuad — COMMENTED** [review]
Many thanks! LGTM

File: cap-ci/pipeline.template (hunk 1 above)
Comment (andreas-kupries) [reply cap-ci/pipeline.template]: Sure, that is no problem at all. The physical ordering is a leftover from the initial logical order. Then I moved the BT towards the end, after the SITS.

Comment (andreas-kupries) [reply cap-ci/pipeline.template]: Done with commit 8f9c24436aa56d082e5c2266fbb33e75c8af7731.

**Review by viccuad — APPROVED** [review]
Many thanks! LGTM

**Comment by viccuad** [comment]
Merging, and fly setting pipeline so it's up-to-date.

**Comment by andreas-kupries** [comment]
Thanks for merging. Moved the associated JIRA ticket to QA.

**Pull request merged**